Cauli/Florets/Inspector/Record/RecordTableViewDatasource.swift

```diff
@@ -67,6 +67,12 @@ extension RecordTableViewDatasource: UITableViewDataSource {
             recordItemCell.textLabel?.text = item.title
             recordItemCell.detailTextLabel?.text = item.description
             recordItemCell.detailTextLabel?.numberOfLines = 15
+
+            if item.value != nil {
+                recordItemCell.accessoryType = .disclosureIndicator
+            } else {
+                recordItemCell.accessoryType = .none
+            }
```

Comment:
This is a small tweak in this PR. Previously all items on the detail page looked like one can tap on them and they would show the actions to share the element or show it in plaintext, but that was not the case. Only some of them did. I added this disclosure indicator to be able to see which ones have details (an underlying value) and which ones don't.